src/index.js

```diff
@@ -15,7 +15,16 @@ export default class MultivariateLinearRegression {
       if (intercept) {
         x.addColumn(new Array(x.rows).fill(1));
       }
-      const beta = new SVD(x, { autoTranspose: true }).solve(y);
+      const xx = x.transpose()
```

Comment:
you can keep `x.transpose()` in a variable to avoid recomputing it in the next statement